src/Estimators/OneBodyDensityMatrices.h

```diff
@@ -145,6 +147,11 @@ class OneBodyDensityMatrices : public OperatorEstBase
   /// current density
   Real rhocur_ = 0.0;
 
+  ///spin related variables
+  Real spcur_;
+  Real dspcur_;
```

Comment:
FYI. The above two uninitialized variables caused failure https://github.com/QMCPACK/qmcpack/actions/runs/8022503383/job/21922019273?pr=4928
Fixed by https://github.com/QMCPACK/qmcpack/pull/4928/commits/1aa51484276f2ea8af92d4c979336fca9abf06b0
Unfortunately compilers didn't warn us in this case.